packages/core/lib/src/widgets/ui/ui.dart

```diff
@@ -3,5 +3,6 @@ export './app_icon_button.dart';
 export './button.dart';
 export './formatted_date_time.dart';
 export './hover_detector.dart';
+export 'navigation/menu_list_item.dart';
```

Comment:
this is missing the leading `.`, but you can leave it as this. if we want to have it in some specific format we should a lint for it.

Reply:
I have changed it to `'./navigation/navigation.dart'` in the next PR